cypress/integration/e2e/sidebar-navigation/index.spec.ts

```diff
@@ -1,37 +1,45 @@
 describe('Sidebar navigation', () => {
   beforeEach(() => {
-    cy.visit('/');
-  });
-
-  it('opens the sidebar disclosures correctly', () => {
-    const gettingStartedButton = '[data-cy=getting-started-button]';
-    const designTokensButton = '[data-cy=design-tokens-button]';
-    const componentsButton = '[data-cy=components-button]';
-    const primitivesButton = '[data-cy=primitives-button]';
-    const layoutButton = '[data-cy=layout-button]';
-    const iconsButton = '[data-cy=icons-button]';
-    const contentButton = '[data-cy=content-button]';
-
-    cy.get(gettingStartedButton)
-      .click()
-      .should('have.attr', 'aria-expanded', 'true');
-    cy.get(designTokensButton)
-      .click()
-      .should('have.attr', 'aria-expanded', 'true');
-    cy.get(componentsButton)
-      .click()
-      .should('have.attr', 'aria-expanded', 'true');
-    cy.get(primitivesButton)
-      .click()
-      .should('have.attr', 'aria-expanded', 'true');
-    cy.get(layoutButton)
-      .click()
-      .should('have.attr', 'aria-expanded', 'true');
-    cy.get(iconsButton)
-      .click()
-      .should('have.attr', 'aria-expanded', 'true');
-    cy.get(contentButton)
-      .click()
-      .should('have.attr', 'aria-expanded', 'true');
+    cy.visit('/', {timeout: 10000});
+  });
+
+  it('opens the getting started disclosure', () => {
+    cy.get('[data-cy=getting-started-button]').click();
+    cy.get('[data-cy=getting-started-list]').should('be.visible');
+  });
+
+  it('opens the tokens disclosure', () => {
+    cy.get('[data-cy=design-tokens-button]').click();
```

Comment:
Can we keep the accessibility tests too `.should('have.attr', 'aria-expanded', 'true');` for these please?